cmd/podman/spec.go

```diff
@@ -53,21 +52,10 @@ func blockAccessToKernelFilesystems(config *createConfig, g *generate.Generator)
 func addPidNS(config *createConfig, g *generate.Generator) error {
```

Comment:
Maybe it 's me on PTO brain, but are we really adding anything with these functions anymore?  Should they be named remove*NS now?  I find it confusing to have an add* function that just basically no-ops or calls a remove function....

Reply:
Perhaps we should change addXXXNS to setupXXXNS.  But I think we can do that in a different PR.